src/test/java/Homework16.java

```diff
@@ -0,0 +1,23 @@
+import org.testng.annotations.Test;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public void Navigation() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example  `public void navigation()`